apps/ai-image-tagging/frontend/src/components/AITagView/AITagView.js

```diff
@@ -118,12 +139,16 @@ export class AITagView extends React.Component {
           ))
         }
       </div>
+      {
+        this.state.unsupportedImageType && !this.state.isMissingImage &&
+        <Note noteType="warning" className={ styles.fileWarning }>Unfortunately, we can only auto-tag PNG/JPG/JPEG file types</Note>
```

Comment:
I wouldn't differentiate between jpg and jpeg. I'd also allow gifs. So "we can only auto-tag PNG, GIF and JPG.

Reply:
The problem actually is that gifs can't be tagged :(

Reply:
okiedokie